### ISSUE TITLE
Package from Rollup is missing some of the components.

### ISSUE THREAD

**Issue by damontgomery** [phase2/outline#79]
As-is, if we get to a certain size (I added outline-alert and outline-modal), Rollup wants to start chunking stuff and it seems to just toss away some random parts of the code or give an error.

I could test this by searching through the `dist/outline.js` file and couldn't find strings like `outline-alert`, but could find `outline-link`.

I think it has something to do with `plugin-multi-entry`. https://www.npmjs.com/package/@rollup/plugin-multi-entry

The closest I got was something like,

```json
input: {
    include: [
      'src/components/**/*.ts',
    ],
    exclude: [
      'src/components/**/*.stories.ts',
      'src/components/**/*.css.lit.ts',
    ],
  },
  output: [
    {
      // Modern bundle.
      dir: 'dist',
      format: 'esm',
      entryFileName: 'outline.js',
    },
  ],
  manualChunks: () => 'outline.js',
```

You can run this with `yarn rollup.watch`.

But this uses the default name of `multi-entry.js` even though I've tried setting `entryFileName` according to the documentation in multiple places.

I'm out of ideas for today, but may be able to try again next week.

**Comment by himerus** [comment]
I can confirm and am investigating at the moment. This is odd behavior. Good catch by setting up/updating the `dev` folder setup. I can validate this on a client project as I attempt to setup things under a client specific folder.

**Comment by himerus** [comment]
Here is a working `rollup.config.js` file I'm using:

```javascript
import summary from 'rollup-plugin-summary';
import { terser } from 'rollup-plugin-terser';
import resolve from '@rollup/plugin-node-resolve';
import replace from '@rollup/plugin-replace';
import multi from '@rollup/plugin-multi-entry';
import typescript from 'rollup-plugin-typescript2';
import svg from 'rollup-plugin-svg';

export default {
  // Output all components to a single exported file.
  input: {
    include: [
      'src/components/**/*.ts',
    ],
    exclude: [
      'src/components/**/*.stories.ts',
      'src/components/**/*.css.lit.ts',
      'src/components/**/*.test.ts',
    ],
  },
  output: [
    {
      // Modern bundle.
      file: 'dist/outline.js',
      format: 'esm',
    },
    {
      // Legacy bundle.
      file: 'dist/outline-legacy.js',
      format: 'cjs',
    },
  ],
  onwarn(warning) {
    if (warning.code !== 'THIS_IS_UNDEFINED') {
      console.error(`(!) ${warning.message}`);
    }
  },
  plugins: [
    replace({ 'Reflect.decorate': 'undefined', 'preventAssignment': true }),
    resolve(),
    typescript(),
    terser({
      ecma: 2020,
      module: true,
      warnings: true,
      mangle: {
        properties: {
          regex: /^__/,
        },
      },
    }),
    summary(),
    multi(),
    svg(),
  ],
};

```

**Comment by himerus** [comment]
Fixed in #80
